Resources/Prototypes/Civ14/Entities/Clothing/entities_clothing_uniform.yml

```
@@ -6776,13 +6776,16 @@
         coefficients:
           Blunt: 1
           Slash: 1
-
+    - type: Butcherable
+      butcheringType: Knife
+      spawned:
+      - id: MaterialLeather1
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)